app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb

```diff
@@ -24,19 +24,17 @@ def ems_inv_to_hashes
     log_header = "MIQ(#{self.class.name}.#{__method__}) Collecting data for EMS name: [#{@ems.name}] id: [#{@ems.id}]"
 
     $aws_log.info("#{log_header}...")
-    AWS.memoize do
```

Comment:
Oh cool...The verification that we don't have N+1 calling is basically done by looking at the VCR cassette.  If you see duplicate calls over and over (and thus the file size of casette becomes enormous), then the calls are not optimized.  Hopefully, this just works, which would be awesome.